src/modules/schnorrsig/tests_impl.h

```diff
@@ -1000,6 +1000,163 @@ static void test_schnorrsig_taproot(void) {
     CHECK(secp256k1_xonly_pubkey_tweak_add_check(CTX, output_pk_bytes, pk_parity, &internal_pk, tweak) == 1);
 }
 
+
+
```

Comment:
nit: We usually have just a single empty line. (Check also in the other files you touched.)